scripts/test.sh

```diff
@@ -2,16 +2,33 @@
 
 echo "Starting Flame Tester"
 echo "---------------------"
+root_dir=$(pwd)
```

Comment:
```suggestion
root_dir="$(dirname "$0")/.."
```

Reply:
I think `root_dir` is only used in one place, so you can just remove this line and do the other suggestion if you want.

Reply:
What is the difference?

About the place, I need it before because the current dir is changed by `cd` commands, so if I use directly on the place I use root_dir, it will not be the value I expect.

Reply:
The difference is that with just `pwd` you have to be in the correct directory for it to work, with `dirname` you can be in any directory. It doesn't matter that you have changed directory since it takes the path of the script.